src/assets/style.scss

```diff
@@ -1,3 +1,6 @@
+// Mixins
+@import "mixins/_form-style";
+
 // Vendor
 @import 'vendor/_normalize.scss';
```

Comment:
Vendor Sass files should come before anything else, but this will already be fixed in [feature/structure](https://github.com/peterxqe/teamfairtrade/tree/feature/structure), so it's not much of a problem.